roles/ceph-osd/tasks/docker/start_docker_osd.yml

```diff
@@ -20,7 +20,7 @@
 # starting the next task
 - name: prepare ceph osd disk
   shell: |   
-    docker run  --net=host \
+    docker run  --net=host --rm \
```

Comment:
I'd like to keep it for debug purpose. If this fails, there is no way to tell what exactly failed.
Agree with @dang's initial thoughts